opencog/embodiment/AtomSpaceExtensions/AtomSpaceUtil.cc

```diff
@@ -1605,14 +1599,13 @@ Handle AtomSpaceUtil::setPredicateValue( AtomSpace& atomSpace,
 Handle AtomSpaceUtil::addPropertyPredicate(AtomSpace& atomSpace,
         std::string predicateName,
         Handle object,
-        TruthValuePtr tv,
-        bool permanent,
-        const Temporal &t)
+	TruthValuePtr tv,
+	bool permanent,
+	const Temporal &t)
```

Comment:
Here is a tab, while the indentation style of that file seems to be 4-whitespace.